### PR TITLE
Video support added in ermrestJS

### PR DESCRIPTION
**Pull request by nitishksahu**
This PR is to support video playback in chaise, for which we needed functionality in ermrestJS supporting custom markdown tag for video

The attributes supported (autoplay, height, loop, muted, preload, src, width)

Only mp4 format as it seems to be supported in all major browsers.

### CODE REVIEW

**Comment by nitishksahu** [comment]
@RFSH the requested changes have been made, this PR is now available for review.

File: js/utilities.js [hunk 1]
```diff
@@ -1262,6 +1262,84 @@
...
+                                    } else if (attrs[0].children[i].type !== 'link_close'){
+                                       captionHTML += md.renderer.renderToken(attrs[0].children,i,{});
+                                    } else {
+                                        break;
+                                    }
+                                }
+                            }
+                            
+                            if(captionHTML.trim().length && flag){
+                                html +=  "<figure><figcaption>"+captionHTML+ "</figcaption>" + videoHTML + videoClass +">"+ srcHTML +"</video></figure>" ;
```
Comment (RFSH) [inline js/utilities.js]: Your implementation of `figcaption` is different than the other tags. Take a look at those, they have a `pos` attribute. Please change it to be the same behaviour.

**Comment by nitishksahu** [comment]
The requested changes have been made, this PR is available for review.

**Comment by nitishksahu** [comment]
@RFSH Travis is fixed now, can we merge the changes now?

**Comment by RFSH** [comment]
@nitishksahu Sure, go ahead and merge it.

**Pull request merged**